app/src/features/surveys/components/EditDeleteStubCard.tsx

```diff
@@ -0,0 +1,68 @@
+import { mdiPencilOutline, mdiTrashCanOutline } from '@mdi/js';
+import Icon from '@mdi/react';
+import { Card, CardHeader, IconButton } from '@mui/material';
+import { grey } from '@mui/material/colors';
+import React from 'react';
+
+type ISubHeaderData = Record<string, string | number | undefined>;
+
+interface EditDeleteStubCardProps {
+  header: string;
+  subHeaderData: ISubHeaderData;
+  onClickEdit?: () => void;
+  onClickDelete?: () => void;
+}
+
+export const EditDeleteStubCard = ({ header, subHeaderData, onClickEdit, onClickDelete }: EditDeleteStubCardProps) => {
+  const formatSubHeaderString = (subHeaderData: ISubHeaderData) => {
+    const formatArr: string[] = [];
+    const entries = Object.entries(subHeaderData);
+    entries.forEach(([key, value]) => {
+      if (value == null || value === '') {
+        return;
+      }
+      formatArr.push(`${key} • ${value}`);
```

Comment:
Nit: prefer [HTML entities](https://www.w3schools.com/html/html_entities.asp) over Unicode.

Reply:
(If you can't get this to work very easily, just ignore this comment)

Reply:
Yea the solution for this is a bit more complex than I think its worth. Just gonna keep it like this for now